src/main.js

```diff
@@ -1,33 +1,40 @@
-import * as Sentry from "@sentry/svelte";
+// import * as Sentry from "@sentry/svelte";
```

Comment:
Can you remove the commented out code before deploying

Reply:
Done